packages/devtools/lib/src/memory/memory.dart

```diff
@@ -138,27 +135,13 @@ class MemoryScreen extends Screen with SetStateMixin {
             ]),
         ]),
       memoryChart = MemoryChart(this, memoryController)..disabled = true,
-      div(c: 'section'),
       tableContainer = div(c: 'section overflow-auto')
         ..layoutHorizontal()
         ..flex(),
     ]);
 
     _pushNextTable(null, _createHeapStatsTableView());
 
-    bool splitterConfigured = false;
-
-    if (!splitterConfigured) {
-      split.flexSplit(
-        [memoryChart, tableContainer],
-        horizontal: false,
-        gutterSize: defaultSplitterWidth,
-        sizes: [18, 82],
-        minSize: [200, 200],
-      );
-      splitterConfigured = true;
-    }
-
```

Comment:
I was hoping to try having the charts resizable I'm not sure FPS needs it but was going to add it for consistency.  I'm running out of space for events sections (GC, Reset, Snapshots, UI actions, etc.)  Is there a reason why resizing a chart is bad?

As memory capacity/usage increases its hard to see much of anything in small vertically high charts.

Reply:
Hey - sorry for landing this pre-review. My goal with this PR was to keep consistency between pages where possible, and generally to avoid having too much configurability if we can get away with/ reasonable defaults.

Happy to iterate here however; we can re-land the splitter, or discuss the layout off-line.